### DISCUSSION TITLE
A plan to update typescript?

### DISCUSSION THREAD

**Discussion by qqpann** [imbhargav5/rooks#1088] [General]
Current latest stable version is 4.7.
Is there an update plan or update strategy? @imbhargav5 

The current typescript is 4.3 and upgrading the version would make contribution easier.

**Comment by qqpann** [comment]
https://github.com/imbhargav5/rooks/issues/1058
Looks like it is awaiting update by bot.

**Reply by imbhargav5** [reply]
@qqpann For sure! i will take a look at it tomorrow!

**Reply by qqpann** [reply]
nice 🤗

**Reply by qqpann** [reply]
Btw, seems discussions are not used quite often?

**Reply by imbhargav5** [reply]
True. I somehow never got notifications on discussions.
